From a2cd906c36a04c8ce5a0095ae53892902b30d728 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Wed, 28 Dec 2022 17:51:27 +0800
Subject: [PATCH] CMakeLists.txt: disable USE_NGHTTP2

nghttp2 depends on cmake-native to build, to break circular
dependency, disable nghttp2.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Changqing Li <changqing.li@windriver.com>

Adjust the patch to apply on top of v3.28.3.

Signed-off-by: Trevor Gamblin <tgamblin@baylibre.com>

Adjust the patch to apply on top of v3.30.1.

Signed-off-by: Trevor Gamblin <tgamblin@baylibre.com>
---
 Utilities/cmcurl/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Utilities/cmcurl/CMakeLists.txt b/Utilities/cmcurl/CMakeLists.txt
index b766aee425..34ae7d2a39 100644
--- a/Utilities/cmcurl/CMakeLists.txt
+++ b/Utilities/cmcurl/CMakeLists.txt
@@ -94,7 +94,7 @@ set(USE_ECH OFF)
 set(USE_HTTPSRR OFF)
 set(USE_LIBIDN2 ON)
 set(USE_LIBRTMP OFF)
-set(USE_NGHTTP2 ON)
+set(USE_NGHTTP2 OFF)
 set(USE_NGTCP2 OFF)
 set(USE_OPENSSL_QUIC OFF)
 set(USE_QUICHE OFF)
-- 
2.39.2

